imports/client/components/ChatPeople.tsx

```diff
@@ -302,20 +291,20 @@ const ChatPeople = ({
         deafened: false,
       };
 
-      const prevEffectiveMuteState = prevState.muted || prevState.deafened;
-      const nextEffectiveMuteState = nextState.muted;
-
-      // Update gain if needed
-      if (prevEffectiveMuteState !== nextEffectiveMuteState) {
-        updateGain(nextEffectiveMuteState);
-      }
+      // Pausing or unpausing the consumer in mediasoup should be sufficient to
+      // stop transmitting audio, but just to be sure, we disable the capture
+      // stream here as well.
+      audioState?.mediaSource?.getTracks().forEach((t) => {
```

Comment:
shouldn't `audioState` always be truthy here?  we shouldn't need the `?.` unless I'm misreading something

Reply:
Yeah it is, I thought that one of the typescript-aware linters would have complained about that. /shrug